openmc/geometry.py

```diff
@@ -42,7 +42,7 @@ def root_universe(self, root_universe):
 
         self._root_universe = root_universe
 
-    def get_offset(self, path, filter_offset):
+    def get_offset(self, path):
```

Comment:
Along the same lines as my comment below - perhaps `get_offset` should be renamed to `get_cell_instance_id(...)` or `get_distribcell_id(...)`? I know your PR addresses a separate issue, this is just a thought that came to mind as I revisit this code in the diff here.